docs/alternator/alternator.md

```diff
@@ -50,6 +50,13 @@ these default locations can overridden by specifying
 `--alternator-encryption-options keyfile="..."` and
 `--alternator-encryption-options certificate="..."`.
 
+By default, Scylla saves a snapshot of deleted tables. But Alternator does
+not offer an API to restore these snapshots, so these snapshots are not useful
+and waste disk space - deleting a table does not recover any disk spaced.
```

Comment:
s/spaced/space/

Reply:
Thanks. Pushed a new version with the typo fixed.